db/fixtures/ae_datastore/ManageIQ/Cloud/Orchestration/Operations/Methods.class/__methods__/available_tenants.rb

```diff
@@ -3,7 +3,7 @@
 #
 tenant_list = {nil => "default"}
 service_template = $evm.root.attributes["service_template"]
-if service_template.orchestration_manager
+if service_template.respond_to?(:orchestration_manager)
```

Comment:
Also, there's very little in the commit message.  Can you clarify why we're changing this?